xml/deploy_cephadm.xml

```diff
@@ -888,10 +847,20 @@ config: OK
   </sect2>
 
   <sect2 xml:id="deploy-cephadm-deploy">
-   <title>Deploy Cluster</title>
+   <title>Update Nodes and Deploy Cluster</title>
    <para>
-    Deploy the previously configured minimal &ceph; cluster by running the
-    following command:
+    Before you deploy the cluster, update all software packages on all nodes:
+   </para>
+<screen>&prompt.smaster;ceph-salt update</screen>
+   <tip>
+    <para>
+     To reboot the nodes after they are updated, append the
+     <option>--reboot</option> option:
+    </para>
+<screen>&prompt.smaster;ceph-salt update --reboot</screen>
```

Comment:
Since https://github.com/ceph/ceph-salt/pull/325 we can also run `ceph-salt reboot` after `ceph-salt update` to reboot nodes if needed:

`ceph-salt update --reboot` and `ceph-salt update + ceph-salt reboot` will do the same.

Is it useful to documment `ceph-salt reboot` too?